charts/ceph-csi-cephfs/templates/provisioner-clusterrole.yaml

```diff
@@ -46,4 +46,12 @@ rules:
     resources: ["persistentvolumeclaims/status"]
     verbs: ["update", "patch"]
 {{- end -}}
+{{- if and .Values.readAffinity .Values.readAffinity.enabled }}
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list","watch"]
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["csinodes"]
+    verbs: ["get", "list", "watch"]
```

Comment:
why do we need this in provisioner role? i don't see any codes changes in controller part here.

Reply:
Needed this as we are fetching node labels in driver.go

```go
nodeLabels, err = k8s.GetNodeLabels(conf.NodeID)
if err != nil {
	log.FatalLogMsg(err.Error())
}
```

had this error 👇
```
driver.go:115] failed to get node "m1" information: nodes "m1" is forbidden: User "system:serviceaccount:csi:csi-cephfsplugin" cannot get resource "nodes" in API group "" at the cluster scope
```

Reply:
For that you need it in plugin-clusterrole not in the provisioner-clusterrole